src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs

```diff
@@ -192,6 +192,9 @@ private static IEnumerable<object[]> ProtocolMismatchData()
               X509Chain chain,
               SslPolicyErrors sslPolicyErrors)
         {
```

Comment:
Consider renaming the method, since it's not entirely AllowAny anymore.  It's sort of `ExpectNoClientCertificate`.